<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fakescript ide</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="txtedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="overwriteMode">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="output">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="memview"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="routinueview"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="callstackview"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="byteview"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionView_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_QtPad"/>
   </widget>
   <widget class="QMenu" name="menuFIND">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFIND"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGoto_Line"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionRun"/>
    <addaction name="actionRun_Args"/>
    <addaction name="actionStep"/>
    <addaction name="actionBreakpoint"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFIND"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFIND"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_QtPad"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>250</horstretch>
     <verstretch>250</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>200</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="btnfindprev">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnfindnext">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtfind">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>250</horstretch>
     <verstretch>250</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>525</width>
     <height>200</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QLineEdit" name="txtreplace1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>with</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtreplace2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnreplaceprev">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnreplacenext">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>110</horstretch>
     <verstretch>250</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>200</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QPushButton" name="btngo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>5</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtgo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>50</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>110</horstretch>
     <verstretch>250</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>200</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_31">
    <widget class="QLineEdit" name="txtrunarg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/filenew.png</normaloff>:/image/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/fileopen.png</normaloff>:/image/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/filesave.png</normaloff>:/image/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/editcopy.png</normaloff>:/image/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/editpaste.png</normaloff>:/image/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/editcut.png</normaloff>:/image/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/editundo.png</normaloff>:/image/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/editredo.png</normaloff>:/image/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionView_Help">
   <property name="text">
    <string>View Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_QtPad">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/qt-logo.png</normaloff>:/image/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About QtPad</string>
   </property>
  </action>
  <action name="actionFIND">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/search_b.png</normaloff>:/image/search_b.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="Images/appres.qrc">
     <normaloff>:/image/replace.png</normaloff>:/image/replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGoto_Line">
   <property name="text">
    <string>Goto Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRun_Args">
   <property name="text">
    <string>Run Args</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionBreakpoint">
   <property name="text">
    <string>Breakpoint</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images/appres.qrc"/>
 </resources>
 <connections/>
</ui>
